composer.lock

```diff
@@ -4,7 +4,7 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#composer-lock-the-lock-file",
```

Comment:
Please use `composer update <package>=<version>` instead of bare `composer update`. At least one issue I already observed is new version of `zend-code` behaves differently. We might do full upgrade as a separate task.